### PR TITLE
updated changelog.md for release 1.13.0

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
### What does this PR do?

Release 1.13.0

### What issues does this PR fix or reference?
no ref

### Is it tested? How?
NA

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+### 1.13.0
+- Fix: The default for keyorder [#859](https://github.com/redhat-developer/yaml-language-server/pull/859)
```
Comment (mohitsuman) [inline CHANGELOG.md:2]:
```suggestion
- Fix: The default value for keyorder [#859](https://github.com/redhat-developer/yaml-language-server/pull/859)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+- Fix: older changelog heading levels [#861](https://github.com/redhat-developer/yaml-language-server/pull/861)
+- Fix: hover behavior when indentation not set [#863](https://github.com/redhat-developer/yaml-language-server/pull/863)
+- Add: Run sarif seperately [#866](https://github.com/redhat-developer/yaml-language-server/pull/866)
+- Fix: Formatting YAML file inserts ternary operators [#255](https://github.com/redhat-developer/vscode-yaml/issues/255)
+- Fix: Mark settings readonly [#868](https://github.com/redhat-developer/yaml-language-server/pull/868)
+- Add: Remove Travis CI status [#873](https://github.com/redhat-developer/yaml-language-server/pull/873)
+
+Thanks to Kasama
```
Comment (mohitsuman) [inline CHANGELOG.md:10]: Please add the github usernname too after the contributor's name.

**Review by mohitsuman — CHANGES_REQUESTED** [review]
Few minor changes.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
+### 1.13.0
+- Fix: The default value for keyorder [#859](https://github.com/redhat-developer/yaml-language-server/pull/859)
+- Fix: older changelog heading levels [#861](https://github.com/redhat-developer/yaml-language-server/pull/861)
+- Fix: hover behavior when indentation not set [#863](https://github.com/redhat-developer/yaml-language-server/pull/863)
+- Add: Run sarif seperately [#866](https://github.com/redhat-developer/yaml-language-server/pull/866)
+- Fix: Formatting YAML file inserts ternary operators [#255](https://github.com/redhat-developer/vscode-yaml/issues/255)
+- Fix: Mark settings readonly [#868](https://github.com/redhat-developer/yaml-language-server/pull/868)
+
+Thanks to Roberto Pommella Alegro Kasama
```
Comment (mohitsuman) [inline CHANGELOG.md:9]: Add the GH username as link. In this case it should be https://github.com/Kasama

**Review by mohitsuman — CHANGES_REQUESTED** [review]
Update the link.

**Review by mohitsuman — APPROVED** [review]

**Review by mohitsuman — APPROVED** [review]

**Review by gorkem — APPROVED** [review]

**Pull request merged**